Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM node:8.12-alpine
 
+EXPOSE 3735
```

Comment:
Why is this needed, out of interest? I don't have any problem running the app in Docker, in development, and connecting to  localhost:3735.

Reply:
Removed, can't reproduce why I needed it. I think maybe that HTTP (non-SSL) no longer worked but didn't tell me in any meaningful way, and I discovered both that and tried the EXPOSE at the same time.